lib/services/presets.go

```diff
@@ -851,6 +851,31 @@ func NewPresetHealthCheckConfig() *healthcheckconfigv1.HealthCheckConfig {
 	}
 }
 
+// NewPresetIdentitySecurityViewRole returns a new pre-defined role that
+// allows viewing Teleport Identity Security - access graph resources.
+func NewPresetIdentitySecurityViewRole() types.Role {
+	role := &types.RoleV6{
+		Kind:    types.KindRole,
+		Version: types.V7,
+		Metadata: types.Metadata{
+			Name:        teleport.PresetAccessGraphViewRoleName,
+			Namespace:   apidefaults.Namespace,
+			Description: "Default Teleport Identity Security viewer role",
+			Labels: map[string]string{
+				types.TeleportInternalResourceType: types.PresetResource,
+			},
+		},
+		Spec: types.RoleSpecV6{
+			Allow: types.RoleConditions{
+				Rules: []types.Rule{
+					types.NewRule(types.KindAccessGraph, RO()),
```

Comment:
@tigrato ~ Do we require any other rules for Identity Activity Center ( for a read only user )

Reply:
that's enough